buzzmobile/process/gps_mapper/gps_mapper.py

```diff
@@ -12,25 +12,34 @@
 from std_msgs.msg import String
 
 #Global Variables
-bridge = CvBridge()
 frames = {}
 frames['bearing'] = frames['points'] = frames['location'] = None
```

Comment:
Since things seem a bit messy with the ordering of `x, y`, can you add a comment that specifies what `frames['location']` expects? Is it `x, y`, `y, x`, `y, -x`, etc?